smart_content_cdn.module

```diff
@@ -135,3 +135,81 @@ function smart_content_cdn_page_attachments(array &$attachments) {
   $attachments['#attached']['library'][] = 'smart_content_cdn/gtm_headers';
   $attachments['#attached']['drupalSettings']['gtmHeaders'] = $js_settings;
 }
+
+/**
+ * Implements hook_ssr_evaluate_condition().
+ */
+function smart_content_cdn_ssr_evaluate_condition($condition_id, array $attach_settings) {
+  $condition_evaluation = FALSE;
+  $op = $attach_settings['op'] ?? 'AND';
+
+  switch ($condition_id) {
+    // Geo condition.
+    case 'smart_cdn:geo_continent_code':
+    case 'smart_cdn:geo_country_code':
+    case 'smart_cdn:geo_country_name':
+    case 'smart_cdn:geo_region':
+    case 'smart_cdn:geo_city':
+    case 'smart_cdn:geo_connection_type':
+    case 'smart_cdn:geo_connection_speed':
```

Comment:
I feel like this repeats some code we're using elsewhere. Maybe that can be abstracted and turned into a function to keep things DRY? (It might not be exact, but I know we have a switch somewhere that's doing effectively the same-ish thing.)

Reply:
We could abstract this into a hook or something across the few modules, but this seems okay for now. I'll make a new issue to abstract it into a hook so that both modules could use the same array.